packages/integral_isolates/lib/src/isolate_configuration.dart

```diff
@@ -29,8 +29,21 @@ class IsolateConfiguration<Q, R> {
     );
   }
 
+  IsolateConfiguration<Q, R> copyWith({required Q message}) {
+    return IsolateConfiguration(
+      callback,
+      message,
+      debugLabel,
+      flowId,
+    );
+  }
+
   @override
   String toString() {
-    return 'IsolateConfiguration(message: $message, debugLabel: $debugLabel, flowId: $flowId)';
+    return 'IsolateConfiguration('
```

Comment:
This can probably be written more clear with tripple `'''`?

Reply:
The reason I opted for using single ticks instead is because I wanted the print to be in one line, and not multiple.